source/guides/stacks/dars.rst

```diff
@@ -54,15 +53,22 @@ Launching the Image
       docker pull clearlinux/stacks-dars-mkl
 
 
-#. Once you have downloaded the image, you can run it with
+#. Once you have downloaded the image, you can run it with this command, which will launch the image and drop you into a bash shell inside the container.
 
    .. code-block:: bash
 
-      docker run -it --ulimit nofile=1000000:1000000 --name mkl <name of image>
+      docker run -it --ulimit nofile=1000000:1000000 --name mkl --network host --rm -i -t <name-of-image>
 
-   This will launch the image and drop you into a bash shell inside the container. The :command:`--ulimit nofile=` parameter is required in order to increase the allowed number of open files for the Apache Spark engine.
 
-   If you need to verify the name of the DARS image, you can use the :command:`docker image ls` command to see which images reside on your system.
+   Command Flags
+
+   :command:`--ulimit nofile=` is required in order to increase the allowed number of open files for the Apache Spark\* engine.
+
+   :command:`--name` can be any name of your choice.  This guide is using `mkl`
+
+   :command:`--network host` flag is for simplicity, so the host machine's IP address can be used to access the container.
+
+   If you need to verify the name of the DARS image for the <name-of-image> flag, you can use the :command:`docker image ls` command to see which images reside on your system.
```

Comment:
Can you simplify the final sentence and make it parallel to the others? Something like:
```suggestion
   :command: `<name-of-image>`  Enter :command:`docker image ls` if you need to find the name of the image.
```